### PR TITLE
OSDOCS-3800: Removing RPM install for oc for RHEL 9

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3800

Link to docs preview:
(single page) http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3800-2/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands

Additional information:

### CODE REVIEW

**Review by opayne1 — COMMENTED** [review]
LGTM!

File: modules/cli-installing-cli-rpm.adoc [hunk 1]
```diff
@@ -43,20 +43,16 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
...
-** For Red Hat Enterprise Linux 8:
 +
 [source,terminal]
 ----
 # subscription-manager repos --enable="rhocp-4.11-for-rhel-8-x86_64-rpms"
 ----
++
+[NOTE]
+====
+It is not supported to install the OpenShift CLI (`oc`) as an RPM for {op-system-base-full} 9. You must install the OpenShift CLI for {op-system-base} 9 by downloading the binary.
```
Comment (soltysh) [inline modules/cli-installing-cli-rpm.adoc:54]: I guess same comment as in https://github.com/openshift/openshift-docs/pull/47922/files#r923681274 applies.

Comment (bergerhoffer) [reply modules/cli-installing-cli-rpm.adoc:54]: Unfortunately we can't link to other sections of the docs in our modules :slightly_frowning_face: But in this case, the docs for installing using other methods are right on this same page, so it shouldn't be an issue

**Pull request merged**